src/main/webapp/resources/js/renderCalendar.js

```diff
@@ -695,7 +762,25 @@ function clearEventDialogSingleMulti(){
     $('#color-select').val("#1ba1e2");
     $('#event-title').val("");
     $('#description').val("");
+}
 
+function buildTableMonthly() { //generating table for DAYS_IN_MONTH day
+    var tableHTML = '';
+    var dayNum = 0;
+    var monthlyTable = document.getElementById("monthly-days");
+    while (dayNum<=DAYS_IN_MONTH) {
+        tableHTML+="<tr>";
+           for(var j = 0; j<DAYS_IN_WEEK;j++) {
+               dayNum++;
+               if (dayNum>DAYS_IN_MONTH) break;
+               tableHTML+=("<td>" + (dayNum) + "</td>");
+           }
+        tableHTML+="</tr>";
+    }
+    monthlyTable.innerHTML=tableHTML;
+    $('table').find('td').click(function() {
+        $(this).toggleClass("active");
```

Comment:
why you mix js and jquery
document.getElementById("monthly-days") -> use $('#monthly-days)